packages/cli/src/commands/deploy/index.ts

```diff
@@ -194,6 +235,21 @@ export default async (client: Client) => {
       );
       return 1;
     }
+
+    const prebuiltBuild = fs.readJsonSync(
+      join(path, '.vercel/output/builds.json'),
+      { throws: false }
+    );
+    if (prebuiltBuild && prebuiltBuild.target !== target) {
+      error(
+        `The ${param(
```

Comment:
We'll want to include an suggestion (i.e. "Please run `vc --prebuilt --target=___` instead") and have an err.sh link to a docs page / FAQ section with a more detailed explanation.

Reply:
I believe we use vercel.link instead of err.sh now

Reply:
I linked this to https://vercel.com/docs/cli#introduction/extended-usage where we can add some more details.

Reply:
How do I make a vercel.link ?

Reply:
I requested access to bit.ly

Reply:
Is there any guidance on how exactly to make these links? Do we add utm params and/or make a "campaign"? I searched Notion, but didn't find anything.

<img width="297" alt="Screen Shot 2022-05-04 at 10 25 57 AM" src="https://user-images.githubusercontent.com/41545/166715643-35982930-ffc7-4ac7-a24d-c7b972fe9ec0.png">

Reply:
There's not really AFAIK, but the nice part is that we can update the target later if we need to, and the link that we use here in CLI won't need to change.